api/notd/manager.py

```diff
@@ -666,6 +668,8 @@ async def update_token_metadata_deferred(self, registryAddress: str, tokenId: st
         await self.tokenManager.update_token_metadata_deferred(registryAddress=registryAddress, tokenId=tokenId, shouldForce=shouldForce)
 
     async def update_token_metadata(self, registryAddress: str, tokenId: str, shouldForce: Optional[bool] = False) -> None:
+        if registryAddress in {'0x495f947276749Ce646f68AC8c248420045cb7b5e'}:
```

Comment:
use constant from model.py

Reply:
or infact, it would be nice if `subCollectionTokenManager` has a `isSubcollectionRegistry` (or better named) function that just returns tru for the contracts it knows about

Reply:
also, do this after the tokenmanager call below